### DISCUSSION TITLE
Is there a way to use hyperswitch without PCI compliance?

### DISCUSSION THREAD

**Discussion by jkazuo55** [juspay/hyperswitch#3802] [General]
I do not want to recover sensitive payment information to avoid PCI Compliance

**Comment by manojradhakrishnan** [comment]
PCI compliance is mandatory if any of the one activity happens in your stack - collection, processing and storage of card information. 

However, there are multiple levels of PCI compliance depending on the number of card transactions handled. It is explained here: https://docs.hyperswitch.io/hyperswitch-open-source/going-live/pci-compliance/its-no-rocket-science

As of today, if you do not wish to manage PCI compliance on your own, the only solution is using Hyperswitch Cloud.

**Reply by jkazuo55** [reply]
thanks!
